notebooks/6_Putting_it_All_Together.ipynb

```diff
@@ -0,0 +1,1002 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# np.savez('soap_file.npz', X=X_raw, Y=Y_raw, property_names=properties, target=target)\n",
+    "# np.savez('soap_hypers.npz', **hypers)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Skip Computing SOAPs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# # Atomistic structure manipulation\n",
+    "# from ase import io\n",
+    "\n",
+    "# # Import SOAP vectors from numpy file\n",
+    "# soap_file = 'soap_file.npz'\n",
+    "# data = np.load(soap_file)\n",
+    "# X_raw = data['X'] # soap vectors (n_samples x n_features)\n",
+    "# Y_raw = data['Y'] # property vectors (n_samples x n_properties)\n",
+    "# target = data['target']\n",
+    "# properties = data['property_names'] # names of your properties\n",
+    "\n",
+    "# hyper_file = 'soap_hypers.npz'\n",
+    "# hypers = np.load(hyper_file)\n",
+    "\n",
+    "# num_features = X_raw.shape[1]\n",
+    "\n",
+    "# # If you end up wanting a chemiscope, you'll need to import your frames as well\n",
+    "# ase_file = \"../datasets/CSD-1000R.xyz\"\n",
+    "# frames = np.asarray(io.read(ase_file, ':'), dtype=object)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Split into training, validation, and testing subset and standardize"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Do a random, test-train split"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.model_selection import train_test_split\n",
+    "\n",
+    "f_train = 0.7\n",
+    "f_test = 0.3\n",
+    "\n",
+    "i_train, i_test = train_test_split(np.arange(X_raw.shape[0]), test_size=f_test)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# np.savez('split_file.npz', i_train=i_train, i_test=i_test)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Skip test-train split"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# # Import testing and training indices from numpy file\n",
+    "\n",
+    "# index_file = 'split_file.npz'\n",
+    "\n",
+    "# data = np.load(index_file)\n",
+    "# i_train = data['i_train']\n",
+    "# i_test = data['i_test']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Scale and Center data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from skcosmo.preprocessing import StandardFlexibleScaler as SFS\n",
```

Comment:
I don't like renaming imports, it makes it harder for a reader of the code to find the corresponding documentation, since they have to find the actual name first.